modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/RestResponseEntityExceptionHandler.java

```diff
@@ -16,13 +16,15 @@
 package com.expedia.adaptivealerting.modelservice.web;
 
 import com.expedia.adaptivealerting.modelservice.repo.ItemExistsException;
+import lombok.Generated;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.context.request.WebRequest;
 import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
 
 @ControllerAdvice
+@Generated //(exclude from code coverage)
```

Comment:
I don't see why we would exclude this from coverage. It should be easy enough to test. Not super useful granted but I don't see excluding it.

Reply:
Updated, but mock for final method is not supported.